features/facebookImport/src/model/importers/posts-importer.js

```diff
@@ -0,0 +1,23 @@
+import MultipleFilesImporter from "./multiple-files-importer";
+import { MissingPostFilesException } from "./utils/failed-import-exception";
+import { removeEntryPrefix } from "./utils/importer-util";
```

Comment:
The more I see this, the more I think we should have a podURI class or something like that that would include as a method (or attribute) the entry with the prefix removed. Probably out scope, but worth considering anyway, possibly for later.

Reply:
Yes would be a good idea. Now we model all ids/URIs just as simple strings which has limitations. We can create an issue.